MBBSEmu/HostProcess/HostRoutines/FsdRoutines.cs

```diff
@@ -627,19 +687,57 @@ private void ExitingFullScreenDisplay(SessionBase session)
             session.CurrentModule.Memory.SetZero(fsdscbStruct.newans, 0x800);
             session.CurrentModule.Memory.SetArray(fsdscbStruct.newans, Encoding.ASCII.GetBytes(answersString));
```

Comment:
+ null terminate?

Reply:
The pattern I'm using for writing Spans to these arrays is setting zero to the destination array prior to writing a ASCII result. It's probably lazy -- but guarantees it'll be null terminated 😛